helm_deploy/hmpps-notifications-alerts-vsip/Chart.yaml

```diff
@@ -2,11 +2,11 @@ apiVersion: v2
 appVersion: '1.0'
 description: A Helm chart for Kubernetes
 name: hmpps-notifications-alerts-vsip
-version: 0.2.1
+version: 0.2.2
 dependencies:
   - name: generic-service
     version: "3.6"
     repository: https://ministryofjustice.github.io/hmpps-helm-charts
   - name: generic-prometheus-alerts
-    version: 1.10.0
+    version: 1.11.3
```

Comment:
Just for info we don't usually go with patch versions of these and stick to major.minor 🙂